src/rpc/methods/chain.rs

```diff
@@ -869,6 +869,58 @@ quickcheck::quickcheck! {
     }
 }
 
+pub async fn get_parent_receipts<DB: Blockstore + Send + Sync + 'static>(
```

Comment:
Can you use `Receipt::get_receipt` here? If not, let's explain why in the comments.

Reply:
`Receipt::get_receipt` return some `Option<Receipt>` but we need to build a `Vec<ApiReceipt>`.